app/src/components/RequestCard/index.js

```diff
@@ -0,0 +1,271 @@
+import CardActions from '@material-ui/core/CardActions';
+import Collapse from '@material-ui/core/Collapse';
+import Avatar from '@material-ui/core/Avatar';
+import IconButton from '@material-ui/core/IconButton';
+import Typography from '@material-ui/core/Typography';
+import { red } from '@material-ui/core/colors';
+import DoneIcon from '@material-ui/icons/Done';
+import CloseIcon from '@material-ui/icons/Close';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import ProfilePage from '../../containers/ViewProfile'
+import ConstantsList from '../../config_constants';
+import AccountCircleIcon from '@material-ui/icons/AccountCircle';
+import WarningIcon from '@material-ui/icons/Warning';
+import Tooltip from '@material-ui/core/Tooltip';
+import Moment from 'moment';
+import Box from '@material-ui/core/Box';
+import Chip from '@material-ui/core/Chip';
+import ZoomInIcon from '@material-ui/icons/ZoomIn';
+import Dialog from '@material-ui/core/Dialog';
+
+
+
+const useStyles = theme => ({
+    root: {      
+      width: '99%',
+      height: '99%',
+    },
+    media: {
+      height: '40px',
+      width: '40px',
+       // 16:9
+    },
+    expand: {
+      transform: 'rotate(0deg)',
+      marginLeft: 'auto',
+      transition: theme.transitions.create('transform', {
+        duration: theme.transitions.duration.shortest,
+      }),
+    },
+    expandOpen: {
+      transform: 'rotate(180deg)',
+    },
+    avatar: {
+      backgroundColor: red[500],
+    },
+    summaryDetails: {
+      height: '100px'
+    },
+    chip: {
+      margin: '2px',
+    },
+});
+  
+class RequestCard extends Component {
+
+    constructor(props) {
+        super(props);
+        this.handleOnYesClick = this.handleOnYesClick.bind(this);
+        this.handleOnNoClick = this.handleOnNoClick.bind(this);
+        this.handleOnError = this.handleOnError.bind(this);    
+        this.state = 
+        {
+            detailProfileOpen: false,
+            portOption:ConstantsList.PORT_IN_USE,
+            showDefaultAvatar: false,
+            detailsExpanded: false,
+            menuOpen: false,
+        };
+    }
+
+   
+    handleExpandClick = () => {
+        const expanded = this.state.detailsExpanded;
+        this.setState({
+            detailsExpanded: !expanded
+        });      
+    }
+
+    renderAvatar = (classes) => {
+        const defaultAvatar = (<CardMedia
+          className={classes.Avatar}
+          component={AccountCircleIcon}
+          children=" " />);
+        
+        const UserAvatar = (<CardMedia
+          className={classes.media}
+              component="img"
+              onError={this.handleOnError}
+              image={window.location.protocol + '//' + window.location.hostname + this.state.portOption + '/api/v1/avatar/getAvatar/' + this.props.user.email}
+        />);
+  
+        if(this.state.showDefaultAvatar) return defaultAvatar;
+        else return UserAvatar
```

Comment:
`else` is not necessary here, just `return UserAvatar` will do because if some case it goes to `if` it will return and stop execution of the function anyway.
It is considered a good practice by popular style guides, to have conditional statements (even if it is a one liner) in curly braces blocks.

Overall, this `renderAvatar` could be refactored where, you could return the element (`defaultAvatar`) at the beginning of the function under the `if` condition and just return `UserAvatar` outside of the if. No need to assign to variables.